tests/resources/functional/snowflake/core_engine/lca/lca_homonym.sql

```diff
@@ -0,0 +1,27 @@
+-- snowflake sql:
+select
+  ca_zip
+from (
+  SELECT
+    substr(ca_zip,1,5) ca_zip,
+    trim(name) as name,
+    count(*) over( partition by ca_zip)
+  FROM
+    customer_address
+  WHERE
+    ca_zip IN ('89436', '30868'));
+-- databricks sql:
+SELECT
+  ca_zip
+FROM
+SELECT
+  SUBSTR(ca_zip,1,5) AS ca_zip,
+  TRIM(name) AS name,
+  COUNT(*) OVER (
+    PARTITION BY
+      SUBSTR(ca_zip,1,5)
+  )
+FROM
+  customer_address
+WHERE
+  SUBSTR(ca_zip,1,5) IN ('89436', '30868');
```

Comment:
Oooh. With which projection should the `WHERE` be associated? I _think_ it's okay?

Reply:
Indeed. In the absence of parenthesis, the WHERE clause is associated with the closest SELECT